src/shared/reboot-util.c

```diff
@@ -110,14 +110,14 @@ int reboot_with_parameter(RebootFlags flags) {
 }
 
 int shall_restore_state(void) {
```

Comment:
Could you install cache for this? rfkill calls this multiple times.

Reply:
-> #28809